### ISSUE TITLE
Highlight last character (python)

### ISSUE THREAD

**Issue by smacker** [bblfsh/dashboard#68]
Happens only for python. For java works fine.

When I hover on node it doesn't highlight last character (for example `fizzbuz` instead of `fizzbuzz`)
![screen shot 2017-11-30 at 5 56 09 pm](https://user-images.githubusercontent.com/406916/33431882-d3e76a96-d5f7-11e7-84a3-ed6a79fbacdf.png)

**Comment by dpordomingo** [comment]
If it's working for Java, but not for python, it may be a problem with the driver, that's not sending the right end position. We should confirm that

**Comment by smacker** [comment]
I have discovered that java&python drivers work different, both ways actually make sense but it should be consistent.
So the change will be done in driver.

I'm keeping it open for now just in case.

**Comment by dpordomingo** [comment]
whoooa! :)
Could you find the line (or docs or something linkable) where both drivers differ?

**Comment by smacker** [comment]
Yes. But it doesn't make sense to add links here. Babelfish team is aware of the problem:

> Juanjo Álvarez [3:24 PM] 
> Yep, native AST differences, we must define ASAP the right ending position column to fix the drivers with the other behavior
> [3:26] 
> Al write it clearly in the docs

**Comment by dpordomingo** [comment]
Is there any issue on the other side?Maybe this one https://github.com/bblfsh/python-driver/issues/142 ?
It would be useful to keep everything documented/updated/related, won't it?

**Comment by smacker** [comment]
the problem isn't python driver. There is no definition for ending position right now. So drivers can implement it differently (I didn't test what drivers for other languages return but it can be the same as java or python or something different as well).
Right now it's unclear where to create an issue. Most probably in bblfsh/documentation or maybe bblfsh/sdk or bblfsh/bblfshd. As soon as we create an issue I'll link it here.

**Comment by smacker** [comment]
Here you go: https://github.com/bblfsh/documentation/issues/130

**Comment by dpordomingo** [comment]
Great!
Now you confirmed the problem is in bblfsh we should maybe label this issue as `blocked` (closing it might be understood "there is no longer problem with this"). What do you think?
